src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/BackendTimeSlot.kt

```diff
@@ -0,0 +1,16 @@
+package com.atlassian.performance.tools.jiraactions.api
+
+import java.time.Duration
+import java.time.Instant
+
+data class BackendTimeSlot internal constructor(
```

Comment:
Personally I'd prefer the data class methods to be explicit when they are becoming API. Otherwise it's easy to unintentionally miss this when changing the class.